src/completers/git.zsh

```diff
@@ -554,6 +554,69 @@ _fzf_complete_git() {
         return
     fi
 
+    if [[ $subcommand = stash ]]; then
```

Comment:
Can you move this block of code after `[[ $subcommand = show ]]`? I want to maintain these conditions (loosely) alphabetically ordered.

Reply:
It's ok, but the order of other commands is also broken like `reset` > `commit` > `add` > `fetch`.

Reply:
We might want to fix them sooner or later...